mozregression/build_data.py

```diff
@@ -402,6 +404,9 @@ def __init__(self, fetch_config, start_rev, end_rev):
 
         self.info_fetcher = BuildFolderInfoFetcher(None, None)
 
+    def was_empty(self):
```

Comment:
Personally I don't see the point of the wrapper method here -- can you not just read/write was_empty directly?

I'd at least make it a property, so you don't have to call the function.